app/views/submissions/reader_review.html.erb

```diff
@@ -1,3 +1,9 @@
+<% content_for :header do %>
+  <%= render HeaderComponent.new(current_user: @current_user,
+                                 title: 'View dissertation or thesis',
+                                 subtitle: 'Reader review and approval') %>
```

Comment:
Worth localizing this text or nah?

Reply:
I'm localizing in cases where a string is used more than once. In this case, this is the only use.